ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -229,6 +230,129 @@ export const ConfirmDelete = ({onCancel, onConfirm}) => {
   </Fragment>;
 };
 
+export const ConfirmDeleteUnattachedPD = ({onConfirm, onCancel}) => {
+  const [deleting, setDeleting] = useState(false);
+
+  return <Fragment>
+    <div style={{display: 'flex'}}>
+      <ClrIcon style={{color: colors.warning, gridColumn: 1, gridRow: 1}} className='is-solid'
+               shape='exclamation-triangle' size='20'/>
+      <h3 style={{...styles.baseHeader, ...styles.bold, gridColumn: 2, gridRow: 1}}>Delete environment options</h3>
+    </div>
+
+    <div style={styles.confirmWarning}>
+      <h3 style={{...styles.baseHeader, ...styles.bold, gridColumn: 1, gridRow: 1}}>
+        <label ><input type= 'radio' checked={deleting === true} onChange={() => setDeleting(true)}/>
+          Delete persistent disk</label></h3>
+      <p style={{...styles.confirmWarningText, gridColumn: 1, gridRow: 2}}>
+        Deletes your persistent disk, which will also delete all files on the disk.
+      </p>
+      <p style={{...styles.confirmWarningText, gridColumn: 1, gridRow: 3}}>
+        If you want to permanently save some files from the disk before deleting it,
+        you will need to create a new cloud environment to access it. </p>
+    </div>
+      <div>
+        <div>If you want to save some files permanently, such as input data, analysis outputs,
+          or installed packages,<a href= 'https://support.terra.bio/hc/en-us/articles/360026639112'>move them to the workspace bucket.</a>
```

Comment:
We generally link to our own Zendesk, not Terra. I don't know off-hand whether there's an article for this. You can check with Yonghao or Shimon to check prod for this, or else just remove the link.

Reply:
I can ask Lou/Shimon. I saw that page in mock but Lou did not give us that link's url

Reply:
Add zendesk article here: https://github.com/all-of-us/workbench/blob/master/ui/src/app/utils/zendesk.ts#L55 for each env. 
To get the ID, login to workbench, search for the articles, the id is in the url. you might not have access to all env, here are they: 
sandbox: 360044796611
preprod: 360059002832
prod: 360040043072